gradle/scripts/log4j-test.properties

```diff
@@ -24,7 +24,7 @@ appenders = console
 appender.console.type = Console
 appender.console.name = STDOUT
 appender.console.layout.type = PatternLayout
-appender.console.layout.pattern = %d [%level] %logger{1.} - %m%n%ex{full}
+appender.console.layout.pattern = %d [%level] %logger{1.} - %m %X%n
```

Comment:
Don't we still want `%ex{full}`, at some future point we may want to play with the filters you can have here....

Reply:
See reference: https://logging.apache.org/log4j/2.x/manual/layouts.html
The %X adds the MDC

Reply:
Added back